rebasebot/bot.py

```diff
@@ -236,7 +188,7 @@ def _identify_downstream_commits(gitwd: git.Repo, source: GitHubBranch, dest: Gi
     return downstream_commits
 
 
-def _do_rebase(gitwd: git.Repo, source: GitHubBranch, dest: GitHubBranch, source_repo: Repository, tag_policy: str,
+def _do_rebase(*, gitwd: git.Repo, source: GitHubBranch, dest: GitHubBranch, source_repo: Repository, tag_policy: str,
```

Comment:
i think i missed this on review before, i see a couple functions you have added `*` as the first arg, i'm guessing this is to allow extra arguments but i'm wondering why we are adding now?

(same in `_init_working_dir` and `run`)

Reply:
Good question. I encountered a newly added linter error about too many positional arguments. 
If the asterisk was in front of the parameter, it would capture all remaining variables. In this case it's alone and specifies zero positional arguments and makes all arguments following it keyword only. 

Good article about it.
https://www.srcecde.me/posts/2024/02/asterisk-and-forward-slash-as-function-parameters-python/